mappings/net/minecraft/structure/NetherFortressGenerator.mapping

```diff
@@ -10,7 +10,12 @@ CLASS net/minecraft/class_3390 net/minecraft/structure/NetherFortressGenerator
 		ARG 6 orientation
 		ARG 7 chainLength
 	CLASS class_3391 BridgeCrossing
+		FIELD field_31559 BLOCK_BOX_SIZE_X I
+		FIELD field_31560 BLOCK_BOX_SIZE_Y I
+		FIELD field_31561 BLOCK_BOX_SIZE_Z I
```

Comment:
I doubt the necessity of `BLOCK_BOX` prefix for all these structures. Since each class is dedicated to one structure, imo `SIZE_X` `SIZE_Y` `SIZE_Z` etc suffices.

Reply:
Alright, I hesitated